web/src/pages/news/articles/[slug].page.tsx

```diff
@@ -23,29 +23,56 @@ import {
 
 import type { GetServerSidePropsContext } from "next";
 
-interface NewsArticlePageProps {
-	story: ISbStoryData<NewsArticleStoryblok>;
+type NewsArticlePageErrorProps = {
+	error: string;
+};
+
+type NewsArticlePageSuccessProps = {
+	story: StoryblokStory<NewsArticleStoryblok>;
 	breadcrumbs?: Breadcrumb[];
-}
+};
 
-export default function NewsArticlePage({
-	story,
-	breadcrumbs,
-}: NewsArticlePageProps): React.ReactElement {
-	const additionalMetaTags = useMemo(
-		() => getAdditionalMetaTags(story),
-		[story]
-	);
+type NewsArticlePageProps =
+	| NewsArticlePageSuccessProps
+	| NewsArticlePageErrorProps;
+
+//TODO: check if moving this out of the component improves performance
```

Comment:
Mind if we add this stuff to the tech debt spreadsheet so that it's all in one place? Just want to make sure it doesn't get buried in the code as these are valid things to check out